src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledDigitalReminderHandler.java

```diff
@@ -97,21 +112,48 @@ protected CompletableFuture<Result<ScheduledNoticeContext>> fetchLoan(
       .thenCompose(r -> r.after(loanPolicyRepository::findPolicyForLoan))
       .thenCompose(overdueFinePolicyRepository::findOverdueFinePolicyForLoan)
       .thenApply(mapResult(context::withLoan))
+      .thenApply(r -> {
+        String servicePointId = r.value().getLoan().getCheckoutServicePointId();
```

Comment:
Calling .value() again. I think this whole section can be extracted to a method (this line and the next line)

Reply:
The entire section is obsolete..